docs/content/latest/integrations/_index.html

```diff
@@ -1,16 +1,15 @@
 ---
-title: Ecosystem integrations using YugabyteDB
-headerTitle: Ecosystem integrations
-linkTitle: Ecosystem integrations
+title: Integrations
+linkTitle: Integrations
 description: Integrate YugabyteDB with Apache Kafka, Apache Spark, JanusGraph, KairosDB, Presto Metabase, WSO2 IS, Hasura
```

Comment:
Some missing from description
headContent - "a few of the ecosystem of"- well we're dropping 'ecosystem ' anyways, so cut or rewrite?

Reply:
Fixed